<!-- Runs the Pepper interface -->
<launch>
    
    <arg name="simulation" default="false"/>
    <arg name="robot_ip" default="169.254.5.105"/>
    <arg name="robot_port" default="9559"/>
    
    <node name="pepper_interface" pkg="pepper_vr_teleop" type="pepper_interface.py" output="screen" unless="$(arg simulation)">
        <param name="robot_ip" value="$(arg robot_ip)"/>
        <param name="robot_port" value="$(arg robot_port)"/>
        <param name="command_duration" value="1.0"/>
        <param name="use_camera" value="false"/>
        <param name="frame_rate" value="30"/>
    </node>
    
    <node name="simulation_interface" pkg="pepper_vr_teleop" type="simulation_interface.py" output="screen" if="$(arg simulation)">
        <param name="motion_time" value="0.2"/>
    </node>    
    
</launch>
